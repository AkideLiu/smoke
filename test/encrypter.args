fixtures/encrypter.rb
fixtures/encrypter
