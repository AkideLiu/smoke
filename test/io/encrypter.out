word
  input:  word
  output: @@ -1 +1 @@
          -xpse
          +jbeq
lines
  input:  one
          two
          three
          four
  output: @@ -1,4 +1,4 @@
          -pof
          -uxp
          -uisff
          -gpvs
          +bar
          +gjb
          +guerr
          +sbhe
color
  input:  this is a [32mgreen[0m word
  output: @@ -1 +1 @@
          -uijt jt b [32mhsffo[0m xpse
          +guvf vf n [32mterra[0m jbeq

3 tests, 3 failures
